src/display.rs

```diff
@@ -528,6 +528,12 @@ pub fn markdown<R: crate::CpuIdReader>(cpuid: crate::CpuId<R>) {
                     "AVX512F: AVX-512 foundation instructions",
                     info.has_avx512f(),
                 ),
+                RowGen::tuple("AVX512-4NNIW: 4NNIW instructions", info.has_avx512_4nniw()),
+                RowGen::tuple(
+                    "AVX512-4FMAPS: 4FMAPS instructions",
+                    info.has_avx512_4fmaps(),
+                ),
+                RowGen::tuple("AMX_BF16: AMX_BF16 instructions", info.has_amx_bf16()),
```

Comment:
should the other two functions also show up here?

Reply:
Depends on what we want. These features are present in only a few intel cpus.
If we want to be super-complete for cli app, we should all three of them. Otherwise, none of them since they are rare.

Reply:
Just removed them since they are very rare

Reply:
Ah I was thinking you can just show everything thats possible there in the CLI

Reply:
My misunderstanding, put them all the new features into the display.